assets/src/edit-story/constants.js

```diff
@@ -27,6 +27,7 @@ export const INSPECTOR_MAX_WIDTH = 280;
 export const PAGE_NAV_PADDING = 60;
 export const PAGE_NAV_BUTTON_SIZE = 40;
 export const PAGE_NAV_WIDTH = PAGE_NAV_PADDING + PAGE_NAV_BUTTON_SIZE;
+export const SAFE_ZONE_HEIGHT = 68;
```

Comment:
Is this the height of the story's system UI (e.g. share icon + progress bar)? Why "safe zone"?

Reply:
Ditto.